generate-manifest.py

```diff
@@ -53,6 +58,17 @@ def edit_manifest(data, arch, branch, runtime_version):
     data['branch'] = branch
     data['runtime-version'] = runtime_version
 
+    # This is nasty
+    gtk_patch = { 'type': 'patch', 'file': 'gtk3-egl-x11.patch' }
+    u = request.urlopen(FREEDESKTOP_MANIFEST_URL)
+    sdk_manifest = json.loads(re.sub(r'(^|\s)/\*.*?\*/', '', u.read().decode('utf-8'), flags=re.DOTALL))
```

Comment:
The less-than-ideal thing about this is it will match on `/*` inside a string, in a path. I think the `\s` should mitigate that, though.

Reply:
This is what FlatHub used, before they moved to some other json-glib-based parser. I honestly don't want to fight against regular expressions for something that's already a nasty hack.

Reply:
Yup. In Flapjack I preprocess the json with json-minify before reading it in, but that's equally ridiculous.